chain/network/src/peer/peer_actor.rs

```diff
@@ -1107,15 +1111,19 @@ impl Handler<QueryPeerStats> for PeerActor {
     fn handle(&mut self, msg: QueryPeerStats, _: &mut Self::Context) -> Self::Result {
         #[cfg(feature = "delay_detector")]
         let _d = DelayDetector::new("query peer stats".into());
+
+        // TODO(#5218) Refactor this code to use `SystemTime`
+        let (sent_per_min, sent_count_per_min) =
+            self.tracker.sent_bytes.get_bytes_per_min_and_count_per_min_and_truncate();
+        let (recv_per_min, recv_count_per_min) =
+            self.tracker.sent_bytes.get_bytes_per_min_and_count_per_min_and_truncate();
+
         PeerStatsResult {
             chain_info: self.chain_info.clone(),
-            received_bytes_per_sec: self.tracker.received_bytes.bytes_per_min() / 60,
-            sent_bytes_per_sec: self.tracker.sent_bytes.bytes_per_min() / 60,
+            received_bytes_per_sec: recv_count_per_min,
```

Comment:
shouldn't you also divide by 60 (as one is per-min, the other one per-sec)

Reply:
Fixed